recipes/r-changepoints/meta.yaml

```
@@ -0,0 +1,100 @@
+{% set name = "r-changepoints" %}
+{% set version = "1.1.0" %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/changepoints_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/changepoints/changepoints_{{ version }}.tar.gz
+  sha256: 73b7e389d8d5590d91ecb369f57b80f605b3c9f231da04be54a650b9b9c593f1
+
+build:
+  merge_build_host: true  # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ compiler('c') }}          # [not win]
+    - {{ compiler('cxx') }}        # [not win]
+    - {{ compiler('m2w64_c') }}    # [win]
```

Include `cxx` for win:

```suggestion
    - {{ compiler('m2w64_c') }}    # [win]
    - {{ compiler('m2w64_cxx') }}  # [win]
```